segments/common/userathost.go

```diff
@@ -0,0 +1,45 @@
+package common
+
+import (
+	"os"
+	"os/user"
+	"strings"
+
+	"github.com/noxer/gops/color"
+	"github.com/noxer/gops/separator"
+)
+
+// AddHost adds the host name of the current machine to the prompt
```

Comment:
Please use the correct function name in the comment

Reply:
Done